### PR TITLE
Codestyle check for Rust

### PR DESCRIPTION
**Pull request by WaDelma**
Implented codestyle checks for Rust. #31

### CODE REVIEW

File: tmc-langs-rust/src/main/java/fi/helsinki/cs/tmc/langs/rust/LintError.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    private final int startLine;
+    private final int startColumn;
+    private final String description;
+    private final String file;
+    private final int endLine;
+    private final int endColumn;
+
+    /**
+     * Creates new error that failing lints generate.
+     * @param file file that the error is in
```
Comment (jamox) [inline tmc-langs-rust/src/main/java/fi/helsinki/cs/tmc/langs/rust/LintError.java]: Add blank line before at-clauses

**Comment by jamox** [comment]
One nitpick, first impressions are good.

**Comment by WaDelma** [comment]
Is this ready to be merged or is there something still missing/wrong?

**Comment by jamox** [comment]
LGTM. Nice work!

**Pull request merged**